_posts/lisetted/2017-05-22-lisette-turtle_hack.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+author: Lisetted
```

Comment:
The capital letter will confuse our program.  You can fix this by finding and modifying this post in our main repository after this pull request is merged.